charts/sysdig/templates/daemonset-node-analyzer.yaml

```
@@ -362,10 +362,105 @@ spec:
               key: no_proxy
               name: {{ template "sysdig.fullname" . }}-benchmark-runner
               optional: true
+{{ if  .Values.nodeAnalyzer.runtimeScanner.deploy }}
+      # Runtime Scanner #
+      - name: sysdig-runtime-scanner
+        image: {{ template "sysdig.image.runtimeScanner" . }}
+        imagePullPolicy: {{ .Values.nodeAnalyzer.runtimeScanner.image.pullPolicy }}
+        securityContext:
+            privileged: true
+        resources:
+{{ toYaml .Values.nodeAnalyzer.runtimeScanner.resources | indent 10 }}
+        # Custom volume mount here
+        env:
+          - name: TMPDIR
+            value: "/tmp"
```

```suggestion
            value: "/tmp/runtimescanner"
```
I can see the `tmp-vol` volume is shared across containers - to avoid improbable but possible collision, we could add a prefix